packages/vscode/src/plugins/nextjs/tests/js-function.test.ts

```diff
@@ -0,0 +1,29 @@
+import { buildVariations } from './__helpers__/buildVariations'
```

Comment:
`js-function.test.ts`, `ts-const.test.ts`, and `ts-function.test.ts` are mostly the same and can probably be moved into a single test file with parameters.

Reply:
I would rather keep them separate

Reply:
What do you think about creaing a single function and just calling it from each file but with different parameters?

Reply:
Ok I'll refactor it :)